libraries/resource_providers/api600/c7000.rb

```diff
@@ -1,4 +1,4 @@
-# (c) Copyright 2018 Hewlett Packard Enterprise Development LP
+# (c) Copyright 2020 Hewlett Packard Enterprise Development LP
```

Comment:
this is not required as we are not changing any code here

Reply:
Updated